src/dropdown/dropdown.js

```diff
@@ -135,6 +155,7 @@ angular.module('ui.bootstrap.dropdown', [])
 
       var toggleDropdown = function(event) {
         event.preventDefault();
+        event.stopPropagation();
```

Comment:
This is a breaking change I believe, since there is a listener on `document` for the click event.